lib/ansible/modules/cloud/google/gcp_forwarding_rule.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/python
+# Copyright 2017 Google Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: gcp_forwarding_rule
+version_added: "2.4"
+short_description: Create, Update or Destory a Forwarding_Rule.
+description:
+    - Create, Update or Destory a Forwarding_Rule. See
+      U(https://cloud.google.com/compute/docs/load-balancing/http/target-proxies) for an overview.
+      More details on the Global Forwarding_Rule API can be found at
+      U(https://cloud.google.com/compute/docs/reference/latest/globalForwardingRules)
+      More details on the Forwarding Rules API can be found at
+      U(https://cloud.google.com/compute/docs/reference/latest/forwardingRules)
+requirements:
+  - "python >= 2.6"
+  - "google-api-python-client >= 1.6.2"
+  - "google-auth >= 0.9.0"
+  - "google-auth-httplib2 >= 0.0.2"
+notes:
+  - Currently only supports global forwarding rules.
+    As such, Load Balancing Scheme is always EXTERNAL.
+author:
+  - "Tom Melendez (@supertom) <tom@supertom.com>"
+options:
+  address:
+    description:
+       - IPv4 or named IP address. Must be of the same scope (regional, global).
+         Reserved addresses can (and probably should) be used for global
+         forwarding rules. You may reserve IPs from the console or
+         via the gce_eip module.
+    required: false
+  forwarding_rule_name:
```

Comment:
Can you make this property "name" since it's already on the forwarding rule module?